### PR TITLE
Javascript Localization Minor Adjustments

### PR DESCRIPTION
**Pull request by shdigitalpy**
I have made some changes to javascript localization after onpage content check with an SEO tool and included structured data json. You will receive details via slack. In my opinion it would be important to include those.

### CODE REVIEW

File: javascript-localization.pug [hunk 1]
```diff
@@ -329,4 +342,30 @@ block content
     window.disableCookiesHook = function () {
       $('.yt-video').hide();
       $('.watch-yt-video-btn').show();
-    };
\ No newline at end of file
+    };
+
+  script(type="application/ld+json").
```
Comment (adrai) [inline javascript-localization.pug:347]: What's this?

**Comment by adrai** [comment]
If possible use always JavaScript, not javascript or Javascript or java-script

File: javascript-localization.pug [hunk 2]
```diff
@@ -249,7 +250,7 @@ block content
             br
             br
             p But there is more.... 
-            p Take a minute and read about <b>automating</b> this process with a <b>smart translation</b> management system below.
+            p Take a minute and read about <b>automating</b> this process with a <b>smart translation</b> management system below. For extensive projects use just a library may too inefficient.
```
Comment (adrai) [inline javascript-localization.pug]: ... using instead of use?

File: javascript-localization.pug (hunk 1 above)
Comment (shdigitalpy) [reply javascript-localization.pug:347]: Google structured data https://developers.google.com/search/docs/guides/intro-structured-data?hl=de

File: javascript-localization.pug [hunk 3]
```diff
@@ -302,24 +303,36 @@ block content
...
+  section.section-white.conclusion#conclusion(style="padding:0")
+   .container
+     .row
+       .col-lg-12
+         .section-heading
+          h2 Now it's your turn
+          p We hope you found everything out of this step-by-step-guide. <b>Best for javascript localization</b> is to start with the <a href="#bestpractices" title="best practices questions">best practices questions</a> we provided above. There are several methods to choose from in order to successfully localize your project with JavaScript: First of all, if you only need to switch the language for an application, it might be easier to use a javascript translation library like i18next or locizify script from us. If you have a huge project with various components to be internationalized and you want to keep up with time, then it might be a good opportunity to use a complete translation management solution like Locize.
```
Comment (adrai) [inline javascript-localization.pug]: We hope **this step-by-step-guide was helpful**...

...like i18next or **our** locizify script...

_if locize is used inside of a sentense (not at the beginning, write it lowercase)_

File: javascript-localization.pug (hunk 2 above)
Comment (shdigitalpy) [reply javascript-localization.pug]: ok

File: javascript-localization.pug [hunk 4]
```diff
@@ -302,24 +303,36 @@ block content
...
+         .section-heading
+          h2 Now it's your turn
+          p We hope you found everything out of this step-by-step-guide. <b>Best for javascript localization</b> is to start with the <a href="#bestpractices" title="best practices questions">best practices questions</a> we provided above. There are several methods to choose from in order to successfully localize your project with JavaScript: First of all, if you only need to switch the language for an application, it might be easier to use a javascript translation library like i18next or locizify script from us. If you have a huge project with various components to be internationalized and you want to keep up with time, then it might be a good opportunity to use a complete translation management solution like Locize. 
+          br
+          div(style="border-left: 5px solid red")
+                  p(style="margin: 1rem;") Since we're offering you a free 14-day trial for Locize, there might be a good chance to test all features including anyway. <a href="https://www.locize.app/register" title="Register now for free">Register now for free</a> trial period and start your best experience with localising an application.
```
Comment (adrai) [inline javascript-localization.pug]: **l**ocize

all included features.

Register now for **the** free

File: javascript-localization.pug [hunk 5]
```diff
@@ -302,24 +303,36 @@ block content
...
+            li Richness (splitting translations to multiple files, string interpolation, plugins for <b>detect language</b>)
+            li Maturity (for any use-case) 
+            li Extensibility (for all modern javascript framework, jquery, laravel and more). 
+
+          br
 
-  //-       br
+          p Now we like to invite you to share that to any developer, project manager or business this would help.
+          
+          p Recommend our step-by-step guide on
```
Comment (adrai) [inline javascript-localization.pug]: step-by-step guide or step-by-step-guide ?

**Pull request merged**